cns/service/main.go

```diff
@@ -1163,34 +1163,24 @@ func InitializeCRDState(ctx context.Context, httpRestService cns.HTTPService, cn
 	poolMonitor := ipampool.NewMonitor(httpRestServiceImplementation, scopedcli, clusterSubnetStateChan, &poolOpts)
 	httpRestServiceImplementation.IPAMPoolMonitor = poolMonitor
 
-	// reconcile initial CNS state from CNI or apiserver.
```

Comment:
this comment is still true right? if so, lets not remove it and also update as "CNI or CNS State or apiserver"

Reply:
it's technically true, but not useful. the decision point for _how_ we're reconciling happens way earlier than this, i think this log is leftover from a much earlier iteration